.github/pull_request_template.md

```diff
@@ -8,25 +8,13 @@ Applicable spec: <link>
 
 <!-- The reason the change is needed -->
 
-### Juju Events Changes
+### Workflow Changes
 
-<!-- Any changes to the juju events being observed (newly added, significantly modified or deleted) -->
-
-### Module Changes
-
-<!-- Any high level changes to modules and why (Service, Observer, helper) -->
-
-### Library Changes
-
-<!-- Any changes to charm libraries -->
+<!-- Any high level changes to workflows and why -->
 
 ### Checklist
 
-- [ ] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
 - [ ] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
-- [ ] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
-- [ ] The documentation is generated using `src-docs`
-- [ ] The documentation for charmhub is updated.
 - [ ] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)
```

Comment:
May I ask what the reasons are for the removal of these points in the checklist?

Reply:
The repository is not likely to include charm code more complex than minimal charms. There're no docstrings and no charmhub docs to nupload

Reply:
Oh right, i assumed this was a charm code PR, thanks!